apis/python/setup.py

```diff
@@ -196,5 +196,11 @@ def run(self):
     },
     python_requires=">=3.7",
     cmdclass={"build_ext": build_ext, "bdist_wheel": bdist_wheel},
-    version=version.getVersion(),
+    use_scm_version={
+        "version_scheme": "guess-next-dev",
+        "local_scheme": "dirty-tag",
```

Comment:
Can we use the defaults here? That way, it will look like `0.5.0a12.dev13+g[short revision sha]` (or `+g[short revision sha].d[datecode]` for dirty checkouts), which is useful for if you are doing work across repos.